fineract-provider/src/test/java/org/apache/fineract/portfolio/account/jobs/executestandinginstructions/ExecutePastAndCurrentInstructionsTest.java

```diff
@@ -0,0 +1,45 @@
+package org.apache.fineract.portfolio.account.jobs.executestandinginstructions;
```

Comment:
Please copy the licence information from an existing file, hence the build is failing due to:
Execution failed for task ':fineract-provider:licenseTest'.